gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=10.0.0
+version=10.1.0
```

Comment:
Technically the lack of this field was a bug... Still we're adding something here, so I bumped the minor version